cql3/query_processor.cc

```diff
@@ -250,12 +327,12 @@ query_processor::query_processor(service::storage_proxy& proxy, database& db, qu
                     sm::make_counter(
                             "reverse_queries",
                             _cql_stats.reverse_queries,
-                            sm::description("Counts number of CQL SELECT requests with ORDER BY DESC.")),
+                            sm::description("Counts the number of CQL SELECT requests with ORDER BY DESC.")),
 
                     sm::make_counter(
                             "unpaged_select_queries",
                             _cql_stats.unpaged_select_queries,
-                            sm::description("Counts number of unpaged CQL SELECT requests.")),
+                            sm::description("Counts the number of unpaged CQL SELECT requests.")),
```

Comment:
it will be good to add a specific internal counter for that as well.

Reply:
Thanks, do you mean to split it like this?
scylla_cql_unpaged_select_queries{shard="x",**keyspace="system"**,type="counter"} 123
scylla_cql_unpaged_select_queries{shard="x",**keyspace="nonsystem"**,type="counter"} 456
Or maybe even this (?) :
scylla_cql_unpaged_queries{shard="x",**keyspace="system",query="select"**,type="counter"} 123

Reply:
first keyspace label as defined in table.cc (see my comment on the previous version) is called ks, use the same name.

don't use keyspace nonsystem.

for the unpaged_select_queries, you can leave the name as it is.
Paging is only relevant to select, no point in adding the extra labels

Reply:
My apologies: the example of `scylla_cql_unpaged_select_queries` was unfortunate. But the general questions still remain:
1. would you like to have label `query` as above?
2. Example from table.cc reads:
`auto ks = keyspace_label(_schema->ks_name());`
Are you suggesting to make separate counters per every KS? The motivation was to isolate the metrics for `system` KS, not **every** KS.
2.a. To avoid counting metrics twice when summing over labels, labels should be "mutually exclusive". What else can it be if not `nonsystem`? The name is wrong?

Reply:
1. a query label can replace the cql command that is part of the name, I think it's better but it's not a must.

2. the problem with nonsystem, is that it's not a real keyspace name and it will block us in the future from adding keyspace names.
I see two options:
a. don't add a keyspace label for non-system, but then don't sum over it.
b. use a different name, that includes only metrics with keyspace, for now, it will only have system, but in the future, we can add more keyspaces to it.